pom.xml

```diff
@@ -219,6 +219,11 @@
     <!-- dependencies are stored in the relevant submodules:
           see main/pom.xml, server/pom.xml and each extension.
     -->
+      <dependency>
+          <groupId>org.apache.commons</groupId>
+          <artifactId>commons-lang3</artifactId>
+          <version>3.4</version>
+      </dependency>
```

Comment:
We should not need to add dependencies here either.

Reply:
Thanks, I will try to remove this unnecessary changes then